### PR TITLE
@jose que lo revise 

### PR DESCRIPTION
**Pull request by pernixgit**

### CODE REVIEW

**Comment by aleks279** [comment]
What's the point of the PR?

You need to add a description of the feature/value you're trying to add. Depending on the feature itself, it could be a brief description or a very long ass description. You may also have to add screenshots, if the feature you're adding includes graphical components.

File: app/views/controller/action.html.erb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<% content_for(:js) do %>
+  <%= javascript_tag do %>
+    $(function()){
+      $(".target").text("<%= some_ruby %>");
+  });
+  <% end %>
+<% end %>
```
Comment (aleks279) [inline app/views/controller/action.html.erb:7]: what's this for?

File: app/views/musicians/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+
+<% @musicians.each do |musician| %>
+  <% @musicians.each do |musician| %>
+  <%= link_to musician.name, musician_path(musican) %>
+<% end %>
+
+<a href="/musicians/1">Paul</a>
+<a href="/musicians/2">Ringo</a>
+<a href="/musicians/3">Jimmy</a>
```
Comment (aleks279) [inline app/views/musicians/index.html.erb:20]: Hard coded links like these are a `no go`.

Rails comes with a `link_to` helper for this.

File: app/views/shared/_footer.html.erb [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+            </div>
+          <div class="col-md-4 footer-nav animated fadeInUp">
+              <h4>Menu —</h4>
+              <div class="col-md-6">
+                    <ul class="pages">
+                        <li><a href="#">Travel</a></li>
+                        <li><a href="#">Nature</a></li>
+                        <li><a href="#">Explores</a></li>
+                        <li><a href="#">Science</a></li>
+                        <li><a href="#">Advice</a></li>
```
Comment (aleks279) [inline app/views/shared/_footer.html.erb:22]: Hard coded links like these are a no go.

Rails comes with a link_to helper for this.

File: app/views/shared/_footer.html.erb [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+                    </ul>
+                </div>
+              <div class="col-md-6">
+                    <ul class="list">
+                        <li><a href="#">About Us</a></li>
+                        <li><a href="#">Contacts</a></li>
+                        <li><a href="#">Terms & Condition</a></li>
+                        <li><a href="#">Privacy Policy</a></li>
```
Comment (aleks279) [inline app/views/shared/_footer.html.erb:30]: Hard coded links like these are a no go.

Rails comes with a link_to helper for this.

File: app/views/shared/_footer.html.erb [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+                    </ul>
+                </div>
+            </div>
+          <div class="col-md-2 footer-social animated fadeInDown">
+              <h4>Follow Us</h4>
+              <ul>
+                  <li><a href="#">Facebook</a></li>
+                  <li><a href="#">Twitter</a></li>
+                  <li><a href="#">Instagram</a></li>
+                  <li><a href="#">RSS</a></li>
```
Comment (aleks279) [inline app/views/shared/_footer.html.erb:40]: Hard coded links like these are a no go.

Rails comes with a link_to helper for this.

File: app/views/shared/_navbar.html.erb [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+                        <li class="dropdown">
+                            <a href="#" class="dropdown-toggle" data-toggle="dropdown">
+                            Dropdown
+                            <b class="caret"></b>
+                            </a>
+                            <ul class="dropdown-menu">
+                                <li><a href="#">Link</a></li>
+                                <li><a href="#">Link</a></li>
+                                <li><a href="#">Link</a></li>
+                                <li><a href="#">Link</a></li>
```
Comment (aleks279) [inline app/views/shared/_navbar.html.erb:28]: Hard coded links like these are a no go.

Rails comes with a link_to helper for this.

File: config/routes.rb [hunk 7]
```diff
@@ -1,6 +1,40 @@
...
+=======
   devise_for :users
   devise_for :admin_users
+
+  resources :clients
+  resources :bands
+  resources :musicians
+  resources :events
+>>>>>>> develop
   # The priority is based upon order of creation: first created -> highest priority.
```
Comment (aleks279) [inline config/routes.rb:38]: This routes file needs to be revised and fixed. If deployed, this would totally break down the app.

File: Gemfile [hunk 8]
```diff
@@ -30,14 +59,27 @@ group :test do
...
+
+
+  # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
+=======
+>>>>>>> develop
   gem 'spring'
+
+
+
+
```
Comment (aleks279) [inline Gemfile:83]: This file needs to be revised and fixed. In this state, it'd be impossible to execute any `bundle` command.

**Comment by cardoce** [comment]
Please remove conflicts.

**Comment by joserv2812** [comment]
why pernixgit?

**Comment by joserv2812** [comment]
what is this @jose que lo revise #4  ?

**Pull request closed without merging**